.github/workflows/security-scan.yml

```diff
@@ -0,0 +1,21 @@
+name: Firebolt Security Scan
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [ main ]
+
+jobs:
+  fossa-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout Code"
+        uses: actions/checkout@v2
+
+      - name: "Security Scan"
+        uses: firebolt-db/action-security-scan@main
+        with:
+          github-key: ${{ secrets.GITHUB_TOKEN }}
+          fossa-key: ${{ secrets.fossaApiKey }}
```

Comment:
Is it possible to have this secret in upper snake case?
Like `secrets.FOSSA_API_KEY`

Reply:
Yeah, once we have the org-wide secrets they'll be upper-case and I will remove this one.

Reply:
Fixed so there's no need to change this later.